utils/runtime-fuzzer/src/lib.rs

```diff
@@ -106,10 +119,23 @@ fn run_impl(data: &[u8]) -> Result<sp_io::TestExternalities> {
     Ok(test_ext)
 }
 
-fn execute_gear_call(sender: AccountId, call: GearCall) -> DispatchResultWithPostInfo {
+fn execute_gear_call(
+    sender: AccountId,
+    call: GearCall,
+    fuzzing_config: &FuzzingConfig,
+) -> DispatchResultWithPostInfo {
+    let allowed_to_spend_value = if fuzzing_config.allow_overspend {
```

Comment:
Otherwise for me seems like bool var 
```suggestion
    let allowed_value_to_spend = if fuzzing_config.allow_overspend {
```

Reply:
_fuzzer related comments_
resolving, a new PR for fuzzer changes will be opened.